taskcluster/ci/config.yml

```diff
@@ -82,4 +82,7 @@ release-promotion:
 private-artifact-prefix: releng/adhoc
 
 scriptworker:
-    scope-prefix: project:adhoc:releng
+    scope-prefix:
+        by-product:
+            mozillavpn: project:releng:signing
+            default: project:adhoc:releng
```

Comment:
Not sure if we want to change the scope here, or if we want to add the adhoc scope to the `script_config.yml` in iscript.

Reply:
We're changing the mac signer's trust domain from `gecko` to `adhoc`, so its scopes will change accordingly. (We'll change the scope prefix in `script_config.yml`)